profiles/fscloud/variables.tf

```
@@ -63,66 +90,57 @@ variable "instance_name" {
 
 variable "tags" {
   type        = list(any)
-  description = "Optional list of tags to be applied to the mongodb instance."
+  description = "Optional list of tags to be added to the MongoDB instance and the associated service credentials (if creating)."
   default     = []
 }
 
 variable "kms_key_crn" {
   type        = string
-  description = "The root key CRN of a Hyper Protect Crypto Service (HPCS) that you want to use for disk encryption. See https://cloud.ibm.com/docs/cloud-databases?topic=cloud-databases-hpcs&interface=ui for more information on integrating HPCS with mongodb instance."
+  description = "The root key CRN of the Hyper Protect Crypto Service (HPCS) to use for disk encryption."
 }
 
 variable "existing_kms_instance_guid" {
-  description = "The GUID of the Hyper Protect Crypto service."
-  type        = string
-}
-
-variable "backup_encryption_key_crn" {
+  description = "The GUID of the Hyper Protect Crypto Service."
   type        = string
-  description = "The CRN of a Key Protect Key to use for encrypting backups. Take note that Hyper Protect Crypto Services for IBM Cloud® Databases backups is not currently supported."
 }
 
 variable "skip_iam_authorization_policy" {
   type        = bool
-  description = "Set to true to skip the creation of an IAM authorization policy that permits all mongodb instances in the provided resource group reader access to the instance specified in the existing_kms_instance_guid variable."
+  description = "Set to true to skip the creation of an IAM authorization policy that permits all MongoDB database instances in the given resource group to read the encryption key from the Hyper Protect instance passed in var.existing_kms_instance_guid."
```

```suggestion
  description = "Set to true to skip the creation of an IAM authorization policy that permits all MongoDB database instances in the resource group to read the encryption key from the Hyper Protect Crypto Services instance. The HPCS instance is passed in through the var.existing_kms_instance_guid variable."
```
Trying to simplify the definition by shortening the sentence.